docs/v1.md

```diff
@@ -167,43 +158,31 @@
 ## OAuthFlows
 - authorizationCode() : `OAuthFlow` | `undefined`
 - clientCredentials() : `OAuthFlow` | `undefined`
-- extension(name) : `any` | `undefined`
-  - name : `string`
+- extensions() : `Extensions`
 - hasAuthorizationCode() : `boolean`
 - hasClientCredentials() : `boolean`
-- hasExtension(name) : `boolean`
-  - name : `string`
 - hasImplicit() : `boolean`
 - hasPassword() : `boolean`
 - implicit() : `OAuthFlow` | `undefined`
 - password() : `OAuthFlow` | `undefined`
 
 ## Operation   
-- binding(bindingProtocol) : `any` | `undefined`
-  - bindingProtocol : `enum{'amqp', 'amqps', 'http', 'https', 'jms', 'kafka', 'kafka-secure', 'mqtt', 'secure-mqtt', 'stomp', 'stomps', 'ws', 'wss'}`
+- bindings() : `Bindings`
 - channels() : `Channel[]`
 - description() : `string` | `undefined`
-- extension(name) : `any` | `undefined`
-  - name : `string`
+- extensions() : `Extensions`
 - externalDocs() : `ExternalDocument` | `undefined`
-- hasBinding(bindingProtocol) : `boolean`
-  - bindingProtocol : `enum{'amqp', 'amqps', 'http', 'https', 'jms', 'kafka', 'kafka-secure', 'mqtt', 'secure-mqtt', 'stomp', 'stomps', 'ws', 'wss'}`
 - hasDescription() : `boolean`
-- hasExtension(name) : `boolean`
-  - name : `string`
-- hasMultipleMessages() : `boolean`
 - hasSummary() : `boolean`
-- hasTags() : `boolean`
 - id() : `string`
 - isApplicationPublishing() : `boolean`
 - isApplicationSubscribing() : `boolean`
 - isClientPublishing() : `boolean`
 - isClientSubscribing() : `boolean`
 - messages() : `Message[]`
-- server() : `Server`
 - servers() : `Server[]`
 - summary() : `string` | `undefined`
-- tags() : `Tag[]`
+- tags() : `Tags`
```

Comment:
Make sure you are consistent either use a wrapper object or a simple array, don't use both IMO. I.e. we use `channel[]`

Reply:
Fixed. Good catch.